libzeth/core/field_element_utils.tcc

```diff
@@ -79,11 +79,11 @@ class field_element_json<libff::Fp_model<n, modulus>>
 {
 public:
     using Field = libff::Fp_model<n, modulus>;
-    static void write(std::ostream &o, const Field &field_el)
+    static void write(const Field &field_el, std::ostream &o)
```

Comment:
I chose so that the "this" argument (the "subject" of the action and the object with type named in the `field_element` function prefix) is the first argument, consistent with the implicit `this` for methods such as `extended_proof::write_json()`.  I think this is a very common C convention for these kinds of functions that "belong" to some type.

Reply:
Got it, that makes sense